trac-env/templates/custom_ticket.html

```diff
@@ -12,9 +12,9 @@ <h2>Please read this first:</h2>
       Report bugs and suggest documentation changes using this ticket tracker.
     </li>
     <li>
-      New feature proposals should first be discussed on the <a href="https://forum.djangoproject.com/c/internals/5">Django Forum</a>.
-      Once feedback has been gathered, you can use this ticket tracker to open a ticket for the new feature.
+      New feature proposals should be created on the <a href="https://github.com/django/new-features/issues/new/choose">new feature idea tracker</a>. The <a href="https://github.com/orgs/django/projects/24">project board view</a> can be used to view existing ideas and their progress..
       Please include links to relevant discussions from the forum (or other sources).
+      If the feature is approved for Django core, a feature ticket may be created here on Trac.
```

Comment:
(nitpick 1) Is "Django core" a well-defined concept? On Trac, "core" is mostly used in the "component" field in opposition to "contrib". Personally I also associate the word with the "core team" which does not exist anymore.

(nitpick 2) What does "approved" mean here exactly? Approved by whom? Per the description of this PR the "new features" project board is experimental, but this proposed sentence makes it sound like it's now a required step in getting a new feature ticket open. Did I misunderstand something?

Reply:
Nitpick 1: I guess we could say `github.com/django/django` instead. I'm not sure if that's better. I'm also open to other suggestions

Nitpick 2: Approved via the new feature tracker or a DEP is what I had in mind. In either case that's currently down to the Steering Council reflecting the community opinion on the proposal. I'm not really sure how much to emphasise or downplay the experimental nature of this process here. I think for the duration of the experiment we're basically treating the new process as *the* way to do this. If we realise after a while that it's also not working, we'll revisit.

Reply:
Thanks for the additional context! I don't have an alternate proposal so I'm going to merge this as-is and we can circle back to it in the future or if we get feedback that this isn't clear.

No need holding back this (positive) change 👍🏻